theme/style.css

```diff
@@ -33,3 +33,30 @@ img {
   background-color: darkgray;
   padding: 75px:
 }
+
+
+ .name {
+  color: green;
+  font: italic; bold;
+  color: green;
+}
+
+h2{
```

Comment:
you're on the right track, but this selector will apply to everyone that's using an `h2`. See if you can make this more specific to your box!